src/core_plugins/metrics/server/lib/vis_data/handle_error_response.js

```diff
@@ -1,5 +1,5 @@
 export default panel => error => {
-  console.log(error);
+  if (error.isBoom) throw error;
```

Comment:
I think it's probably ideal for errors of all types to use boom so that they follow the error path and only trigger `.catch()` handlers. In this case you can dictate the rendered version of the boom error like so:

```js
const result = {};

// populate result

const err = Boom.create(...);
err.output.payload = result // assign the result as the output payload
throw err;
```